turnrest.c

```diff
@@ -31,6 +31,7 @@
 static const char *api_server = NULL;
 static const char *api_key = NULL;
 static gboolean api_http_get = FALSE;
+static uint api_timeout = 10;
```

Comment:
Doesn't look like the default here matters: it's always overwritten in `janus_turnrest_set_backend`.